src/app/stack/card-details/component-details/component-details.component.spec.ts

```diff
@@ -47,7 +47,7 @@ describe ('ComponentDetailsComponent', () => {
         fixture.detectChanges();
     });
 
-    it('should create', () => {
-        expect(component).toBeTruthy();
-    });
+    // it('should create', () => {
+    //     expect(component).toBeTruthy();
+    // });
```

Comment:
Instead of commenting, Please remove if the code is not needed

Reply:
Removed it.